udata/core/spatial/api.py

```diff
@@ -32,6 +35,35 @@
     'size', type=int, help='The amount of suggestion to fetch',
     location='args', default=10)
 
+dataset_parser = api.parser()
+dataset_parser.add_argument(
+    'dynamic', type=bool, help='Append dynamic datasets',
```

Comment:
It might be safe to use flask restplus bool input so the boolean is properly parsed when using `0/1`, `false/true`...